bin/node/executor/tests/submit_transaction.rs

```diff
@@ -70,17 +73,10 @@ fn should_submit_signed_transaction() {
 	t.register_extension(KeystoreExt(keystore));
 
 	t.execute_with(|| {
-		let keys = <SubmitTransaction as SubmitSignedTransaction<Runtime, Call>>
-			::find_all_local_keys();
-		assert_eq!(keys.len(), 3, "Missing keys: {:?}", keys);
-
-		let can_sign = <SubmitTransaction as SubmitSignedTransaction<Runtime, Call>>
-			::can_sign();
-		assert!(can_sign, "Since there are keys, `can_sign` should return true");
-
-		let call = pallet_balances::Call::transfer(Default::default(), Default::default());
-		let results =
-			<SubmitTransaction as SubmitSignedTransaction<Runtime, Call>>::submit_signed(call);
+		let results = Signer::<Runtime, ImOnlineAuthId>::all_accounts()
```

Comment:
I should probably use a pallet `AppCrypto` AuthId here rather than im-online

Reply:
This is a test, I think it is fine. You should use some concrete crypto type and currently I think im-online is the only one atm. I introduce a new one here https://github.com/paritytech/substrate/pull/4517 and actually change these tests a bit. You should have a look.